src/toast/covariance.py

```diff
@@ -16,6 +16,22 @@
 from .pixels import PixelData
 
 
+# These are just wrappers to time the libtoast calls
```

Comment:
Could the decorator not be combined with the pybind11 wrapper?

Reply:
Good point, I will investigate the feasibility.

Reply:
After more investigation, I think doing this within the compiled extension would be much more painful than these trivial decorator-wrappers.  Another option would be to have an middle-layer "libtoast.py" which imported everything from the `_libtoast` compiled extension and selectively decorated some functions.  Then all of the current code which imports `_libtoast` objects would instead import `libtoast` to get those.

That is a tedious, but straightforward change.  I am tempted to just leave these 3 wrappers for now, since this is the only python file where those functions are used.  If we find we want to decorate more compiled code we could implement that mid-level python file in a future PR.  Thoughts?

Reply:
Perfect fine by me.

Reply:
> That is a tedious, but straightforward change. I am tempted to just leave these 3 wrappers for now, since this is the only python file where those functions are used. If we find we want to decorate more compiled code we could implement that mid-level python file in a future PR. Thoughts?

Seems like a good middle ground : no  need to do a lot of modification that will not be useful.